client/src/components/Draftail/CommentableEditor/CommentableEditor.tsx

```diff
@@ -685,14 +673,14 @@ interface CommentableEditorProps {
   fieldNode: Element;
   contentPath: string;
   rawContentState: RawDraftContentState;
-  onSave: (rawContent: RawDraftContentState) => void;
-  inlineStyles: Array<InlineStyle>;
+  onSave: (rawContent: RawDraftContentState | null) => void;
```

Comment:
No runtime change here, it was just the type definition that was off. There shouldn’t be changes needed to the JS implementation of our `serialiseInputValue`.